rustworkx-core/src/centrality.rs

```diff
@@ -75,7 +74,7 @@ use rayon_cond::CondIterator;
 /// [`edge_betweenness_centrality`]
 pub fn betweenness_centrality<G>(
     graph: G,
-    endpoints: bool,
+    include_endpoints: bool,
```

Comment:
I think this ok from a backwards compatibility perspective since in rust the arguments are all passed positionally (this would be breaking in Python so I wanted to just note this and check myself).

Reply:
Oh yeah, that's a good point.